app/src/components/mode-specific/desktop/InterceptTraffic/WebTraffic/TrafficTableV2/ActionHeader.jsx

```diff
@@ -167,6 +173,21 @@ const ActionHeader = ({
     trackRQDesktopLastActivity(TRAFFIC_TABLE.TRAFFIC_TABLE_FILTER_CLICKED);
   };
 
+  const showFilterLogsAlert = (logsToSave, src) => {
+    trackSavingTooManyLogsAlertShown(logsToSave.log.entries.length ?? undefined, src);
```

Comment:
props like `logsCount` and `filteredLogsCount` are passed to this component, can  we use theses instead of using `.length`. 

Same for `canSaveLogsWithoutCrashing` function